IES Specification Docs/Information Exchange Standard r4.2.0.n3

```diff
@@ -2623,7 +2514,7 @@ ies:recurrentPeriodRepresentation rdfs:subPropertyOf ies:attribute .
 ies:recurrentPeriodRepresentation rdfs:domain ies:RecurringPeriod . 
 ies:RecurringPeriod rdf:type rdfs:Class . 
 ies:RecurringPeriod sparx:guid "{986E66AC-9092-410b-88AD-30B86EFC32DD}" . 
-ies:RecurringPeriod rdfs:comment "A PeriodOfTime that is composed of regularly recurring periods of time.ISO8601 is used to represent these periods (recurrentPeriod property), using the blanking technique (e.g. blanking the date to give a daily time). The recurrence can be limited using the recurringFrom and recurringUntil properties Examples:Every Tuesday from 28th August 2018 to 2 October 201813:00 to 14:00 on every day from 27th June 2016 to 2 October 2024" . 
+ies:RecurringPeriod rdfs:comment "A PeriodOfTime that is composed of regularly recurring periods of time.ISO8601 is used to represent these periods (recurrentPeriodRepresentation property), using the blanking technique (e.g. blanking the date to give a daily time). The recurrence can be limited using the startsIn and endsIn properties Examples:Every Tuesday from 28th August 2018 to 2 October 201813:00 to 14:00 on every day from 27th June 2016 to 2 October 2024" . 
```

Comment:
Rules for generating recurring period URI are missing whereas there is explicit advice for ParticularPeriod. Explicit advice here would be highly desirable.

Reply:
I will raise this at the next EWG technical working group meeting